Database/Corrections/Automatic/sodBaseQuests.lua

```diff
@@ -399,6 +399,17 @@ function SeasonOfDiscovery:LoadBaseQuests()
             [questKeys.requiredRaces] = raceIDs.NONE,
             [questKeys.requiredClasses] = classIDs.NONE,
         },
+        [78270] = {
+            [questKeys.name] = "Goblin Lockpicks",
+            [questKeys.startedBy] = {{3988, 4070, 3992, 3993}},
+            [questKeys.finishedBy] = {{4086}},
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 23,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.ROGUE,
+            [questKeys.objectivesText] = {"Find The Horn of Xelthos inside of Shadowfang Keep. Then, return it to C's dead drop near Pyrewood Village."},
+            [questKeys.objectives] = nil,
+        },
```

Comment:
```suggestion
        [78270] = {
            [questKeys.name] = "Goblin Lockpicks",
            [questKeys.startedBy] = nil,
            [questKeys.finishedBy] = {{4086,}},
            [questKeys.requiredLevel] = 1,
            [questKeys.questLevel] = 23,
            [questKeys.requiredRaces] = raceIDs.NONE,
            [questKeys.requiredClasses] = classIDs.ROGUE,
            [questKeys.objectivesText] = {"Find The Horn of Xelthos inside of Shadowfang Keep. Then, return it to C's dead drop near Pyrewood Village."},
            [questKeys.objectives] = {nil,nil,{{210187}}},
        },
```

Just replacing this with the result of the scraper.